src/js/translations/en-US.json

```diff
@@ -21,7 +21,7 @@
   "DASHBOARD.PANEL_HEADING.CPU": "CPU Allocation",
   "DASHBOARD.PANEL_HEADING.MEMORY": "Memory Allocation",
   "DASHBOARD.PANEL_HEADING.DISK": "Disk Allocation",
-  "DASHBOARD.PANEL_HEADING.SERVICES_HEALTH": "Services Health",
+  "DASHBOARD.PANEL_HEADING.SERVICES_HEALTH": "Services Status",
   "DASHBOARD.PANEL_HEADING.TASKS": "Tasks",
   "DASHBOARD.PANEL_HEADING.COMPONENT_HEALTH": "Component Health",
```

Comment:
Not really in the scope of your PR, but I noticed that should be plural "Components Health" for consistency.

Reply:
Yeah not in the scope but I can change that since it's a small change